management/validation.py

```diff
@@ -179,14 +185,16 @@ def check_qcspec_coverage(dataset_data):
     # now try and add each spec
     spec_report = {}
     for spec in qc_specs.values():
-
+        valid_scf_props = check_scf_props(spec)
         try:
+            # remove the scf props so they dont cause issues
+            del spec["scf_properties"]
```

Comment:
What kind of issues? I don't understand why we would delete the `"scf_properties"` key from the spec dict here.

Reply:
just curious about this, I am hitting validation errors on my new submission #220

Reply:
My thinking was as the scf properties are now validated separately on line 188 if they are incorrect they should not then raise an error when validating the method, basis and program so I remove them. The current errors in PRs #220 and #223 are due to the qcsubmit dataset changes which moved the `scf_properties` from the dataset to the qcspecifications.

Reply:
Ah, understood. Thank you! I'll add these details to the inline comment.